src/Tests/dotnet-workload-install.Tests/GivenWorkloadManifestUpdater.cs

```diff
@@ -305,9 +305,10 @@ public void GivenWorkloadManifestUpdateItChoosesHighestManifestVersionInCache()
         }
 
         [Theory]
-        [InlineData("build")]
-        [InlineData("publish")]
-        public void GivenWorkloadsAreOutOfDateUpdatesAreAdvertisedOnRestoringCommands(string commandName)
+        [InlineData("build", false)]
+        [InlineData("publish", false)]
+        [InlineData("run", true)]
+        public void GivenWorkloadsAreOutOfDateUpdatesAreAdvertisedOnRestoringCommands(string commandName, bool commandOptedOut)
```

Comment:
`commandOptedOut` ends up being sort of a double negative.  I would invert the boolean values and call the variable something like `shouldShowUpdateAvailable`.